src/main/java/org/molgenis/vcf/report/AppCommandLineOptions.java

```diff
@@ -82,6 +100,7 @@ static void validateCommandLine(CommandLine commandLine) {
     validateInput(commandLine);
     validateOutput(commandLine);
     validateTemplate(commandLine);
+    validatePed(commandLine);
```

Comment:
add phenotypes validation here?

Reply:
outdated, but not marked as such due to the fix happening on another line